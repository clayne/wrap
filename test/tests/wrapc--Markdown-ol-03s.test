wrapc | /dev/null | -u | md-ol-03s.c | 0
