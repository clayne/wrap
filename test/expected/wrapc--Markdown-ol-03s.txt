  // 1. List item #1.  Second sentence of list item #1.  Third sentence of list
  //    item #1.  Fourth sentence of list item #1.
  // 2. List item #2.
  //
  // Not part of the list.
  //
  // 7. Another list item #7.
  //
  // 8. Another list item #8.
